lib/discover/localpackets.go

```diff
@@ -26,8 +26,8 @@ type Announce struct {
 }
 
 type Relay struct {
-	Address string // max:256
-	Latency int32
+	URL     string `json:"url"` // max:256
```

Comment:
It doesn't hurt.
We eventually can start discovering peers via other peers who can't talk to this peer directly or the discovery server (heading the skype way)